src/groundlight/client.py

```diff
@@ -623,6 +662,7 @@ def add_label(self, image_query: Union[ImageQuery, str], label: Union[Label, str
         else:
             image_query_id = str(image_query)
             # Some old imagequery id's started with "chk_"
+            # TODO: handle iqe
```

Comment:
I'm not sure what you mean here, so I'm concerned that others might not as well, but maybe I'm just out of the loop.

Reply:
Edge detectors return image queries with the prefix `iqe_`, so we need to work on a way to handle that sanely

Reply:
This what the edge server will return, right? This just opens up the sdk to work with the edge

Reply:
got it!

Reply:
Exactly Brandon, just updated the comment to be a bit more clear